### PR TITLE
Bump package versions

### PR DESCRIPTION
**Pull request by alopezsanchez**
Hello!

This PR is related to https://github.com/v8/web-tooling-benchmark/issues/31. First of all, sorry for the delay!

I didn't update `prettier` and `source-map` because they throw errors (https://github.com/prettier/prettier/issues/3925 and https://github.com/webpack/webpack-sources/issues/28#issuecomment-371597972).

Fixes https://github.com/v8/web-tooling-benchmark/issues/31

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "web-tooling-benchmark",
   "description": "JavaScript Benchmark for common Web Developer workloads",
-  "version": "0.3.2",
+  "version": "0.4.0",
```
Comment (mathiasbynens) [inline package.json]: Please don’t bump the project’s version number. That needs to be done by maintainers in a separate commit.

Comment (alopezsanchez) [reply package.json]: Ah, okey, sorry @mathiasbynens. I read it here https://github.com/v8/web-tooling-benchmark/issues/31#issuecomment-357578201

Comment (mathiasbynens) [reply package.json]: https://github.com/v8/web-tooling-benchmark/issues/31#issuecomment-357596714 came later and was then agreed upon. But hey, no worries!

Thanks for contributing this patch and linking to the prettier and source-map issues!
